src/helpers/transport/stream/box_body.rs

```diff
@@ -0,0 +1,53 @@
+use crate::helpers::BytesStream;
+
+use futures::Stream;
+use pin_project::pin_project;
+use std::{
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+type BoxInner = Pin<Box<dyn BytesStream>>;
```

Comment:
Probably should be moved to `BytesStream` definition and renamed to `BoxedBytesStream`?